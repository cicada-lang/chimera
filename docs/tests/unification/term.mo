print find x {
  Equal(x, a.b.c.f(1))
}

print find _ {
  Equal(a.b.f(1), a.b.c.f(1))
}
