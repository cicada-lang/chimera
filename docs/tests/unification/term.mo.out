[a.b.c.f(1)]
[]
